main.pm

```diff
@@ -196,6 +196,11 @@ sub gnomestep_is_applicable() {
     return check_var("DESKTOP", "gnome");
 }
 
+sub snapper_is_applicable() {
+    my $fs = get_var("FILESYSTEM", 'btrfs');
+    return ( $fs eq "btrfs" && get_var("HDDSIZEGB") > 10);
```

Comment:
Since `btrfs` is default, so when `FILESYSTEM` is not set, *get_var* returns `btrfs`, if set, *get_var* returns `FILESYSTEM` value. And then just check if `$fs` is indeed `btrfs` and also disk is larger then 10GB (I guess those are conditions to enabling snapper)